src/migrations/20231017230000-merge-pseudo-fei-goals.js

```diff
@@ -0,0 +1,492 @@
+      -- Relink non-duplicate ARGs
+      -- Delete duplicate ARGs
+      -- Delete donor goals
+
+      -- Update goal templates for fei.
+      CREATE TEMP TABLE goal_merges
+      AS
+      WITH goalmap AS (
+        SELECT -- starting with the 2023 set
+          '01CH011564' grnum,
+          54340 donor_gid,
+          50741 target_gid
+        UNION SELECT '01CH011820', 52284, 50952
+        UNION SELECT '01CH011820', 52416, 50952
+        UNION SELECT '01CH011482', 54341, 52420
+        UNION SELECT '01CH011496', 54342, 51164
+        UNION SELECT '01CH010695', 54343, 51171
+        UNION SELECT '01CH011183', 52285, 50953
+        UNION SELECT '01CH010718', 54344, 51165
+        UNION SELECT '01CH011941', 52286, 50954
+        UNION SELECT '01CH011875', 54345, 51166
+        UNION SELECT '01CH011404', 52287, 50955
+        UNION SELECT '01CH011404', 54360, 50955
+        UNION SELECT '01CH012151', 54351, 54325
+        UNION SELECT '01CH010716', 52288, 50956
+        UNION SELECT '01CH011664', 52289, 50957
+        UNION SELECT '01CH010781', 52290, 50958
+        UNION SELECT '01CH010741', 52291, 50959
+        UNION SELECT '01CH011513', 54346, 51168
+        UNION SELECT '01CH012160', 52292, 50960
+        UNION SELECT '01CH011913', 52293, 52223
+        UNION SELECT '01CH010930', 52295, 50705
+        UNION SELECT '01CH010602', 52296, 50961
+        UNION SELECT '01CH011668', 54347, 51169
+        UNION SELECT '01CH012124', 54348, 51170
+        UNION SELECT '05CH011505', 55653, 55649
+        UNION SELECT '05CH010709', 55651, 55648
+        UNION SELECT '05CH010775', 52460, 52453
+        UNION SELECT '05CH011904', 52461, 52454
+        UNION SELECT '05CH012183', 52462, 52455
+        UNION SELECT '05CH011796', 55655, 54275
+        UNION SELECT '05CH011188', 54308, 54774
+        UNION SELECT '05CH010694', 55650, 55647
+        UNION SELECT '05CH010926', 52319, 50605
+        UNION SELECT '05CH011114', 52318, 50606
+        UNION SELECT '05CH011731', 52317, 50607
+        UNION SELECT '05CH011253', 52463, 52456
+        UNION SELECT '05CH011463', 51508, 52451
+        UNION SELECT '05CH011463', 51654, 52451
+        UNION SELECT '05CH011463', 54307, 52451
+        UNION SELECT '05CH011851', 54316, 55215
+        UNION SELECT '05CH011779', 55654, 53116
+        UNION SELECT '05CH010568', 54002, 51599
+        UNION SELECT '05CH012264', 54767, 56632
+        UNION SELECT '05CH011144', 51892, 51811
+        UNION SELECT '10CH012116', 53611, 53390
+        UNION SELECT '01CH011357', 54349, 54349 -- continuing with the 2093 set
+        UNION SELECT '01CH011073', 54350, 54350
+        UNION SELECT '01CH011423', 54215, 54215
+        UNION SELECT '01CH011874', 52294, 52294
+        UNION SELECT '04CH011857', 55635, 55635
+        UNION SELECT '04CH011025', 56022, 56022
+        UNION SELECT '05CH012028', 55115, 55115
+        UNION SELECT '05CH012316', 56047, 56047
+        UNION SELECT '05CH011333', 54317, 54317
+      ),
```

Comment:
why not write it in the form of: 
SELECT *
FROM (
  VALUES 
    ('01CH011820', 52284, 50952),
    ('01CH011820', 52416, 50952),
    ('01CH011482', 54341, 52420),
    ('01CH011496', 54342, 51164),
    ('01CH010695', 54343, 51171),
    ('01CH011183', 52285, 50953),
    ('01CH010718', 54344, 51165),
    ('01CH011941', 52286, 50954),
    ('01CH011875', 54345, 51166),
    ('01CH011404', 52287, 50955),
    ('01CH011404', 54360, 50955),
    ('01CH012151', 54351, 54325),
    ('01CH010716', 52288, 50956),
    ('01CH011664', 52289, 50957)
) AS data (grnum, donor_gid, target_gid);

Reply:
Just got into the UNION SELECT pattern on Redshift, which didn't fully support the values inserts at one time.